packages/front-end/hooks/useApi.ts

```diff
@@ -1,17 +1,29 @@
 import useSWR from "swr";
 import { useAuth } from "../services/auth";
 
+export const enum Method {
+  GET = "GET",
+  POST = "POST",
+}
```

Comment:
I don't think the `useApi` method should support anything other than `GET`.  This uses `swr`, which caches responses, fires API calls multiple times, etc..  Not safe for POST requests.

POST requests that need to fire on page load should use a `useEffect` hook and the `apiCall` method instead.